force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -253,13 +271,13 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer sumPositiveIntegers() {
         List<Integer> numbers = new List<Integer>{-1, 2, -3, 4, -5, 6}; //DO NOT CHANGE
-
-        // Loop through the list of integers
-            // if the number is negative skip this iteration
-        
-
-        return null; // Replace null with the variable you used to store the result
-    }
+        Integer sum = 0;
+        for(Integer tst : numbers) {
+            if (tst > 0) {
+                sum += tst; }
+            }
+        return sum;
+        }
```

Comment:
Good solution. I would look  for a better name than tst.  

For(Integer integer : numbers)

would work fine